x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/components/fleet_extensions/aws_credentials_form/aws_credentials_form.test.tsx

```diff
@@ -0,0 +1,458 @@
+        packageInfo: {
+          ...defaultProps.packageInfo,
+          name: 'test_package_no_template',
+        },
+        input: {
+          ...defaultProps.input,
+          streams: [
+            {
+              enabled: true,
+              data_stream: defaultProps.input.streams[0].data_stream,
+              vars: {
+                'aws.credentials.type': { value: 'cloud_formation' },
+              },
+            },
+          ],
+        },
+      };
+
+      renderWithProviders(propsWithNoTemplate);
+
+      expect(
+        screen.getByText(/CloudFormation is not supported on the current Integration version/)
+      ).toBeInTheDocument();
+    });
+
+    it('includes read documentation link in CloudFormation setup', () => {
+      const propsWithCloudFormation = {
+        ...defaultProps,
+        input: {
+          ...defaultProps.input,
+          streams: [
+            {
+              enabled: true,
+              data_stream: defaultProps.input.streams[0].data_stream,
+              vars: {
+                'aws.credentials.type': { value: 'cloud_formation' },
+              },
+            },
+          ],
+        },
+      };
+
+      renderWithProviders(propsWithCloudFormation);
+
+      // Should have a documentation link in the setup info
+      expect(screen.getByText('Getting Started')).toBeInTheDocument();
+    });
+
+    it('shows CloudFormation radio button when CloudFormation is supported', () => {
+      renderWithProviders(defaultProps);
+
+      expect(screen.getByRole('radio', { name: 'CloudFormation' })).toBeInTheDocument();
+      expect(screen.getByRole('radio', { name: 'CloudFormation' })).not.toBeDisabled();
+    });
+  });
+
+  describe('Manual Setup', () => {
+    it('renders manual setup components when manual format is selected', () => {
+      renderWithProviders(defaultProps);
+
+      expect(screen.getByTestId('aws-credentials-type-selector')).toBeInTheDocument();
+      expect(screen.getByTestId('externalLink')).toBeInTheDocument();
+    });
+
+    it('passes correct props to AwsCredentialTypeSelector', () => {
+      renderWithProviders(defaultProps);
+
+      const selector = screen.getByTestId('aws-credentials-type-selector');
+      expect(selector).toHaveValue('direct_access_keys');
+      expect(selector).not.toBeDisabled();
+    });
+
+    it('disables credential type selector when disabled prop is true', () => {
+      renderWithProviders({ ...defaultProps, disabled: true });
+
+      const selector = screen.getByTestId('aws-credentials-type-selector');
+      expect(selector).toBeDisabled();
+    });
+
+    it('includes read documentation link with correct URL', () => {
+      renderWithProviders(defaultProps);
+
+      const docLink = screen.getByTestId('externalLink');
+      expect(docLink).toHaveAttribute('href', '/get-started/aws');
+    });
+  });
+
+  describe('Event Handlers', () => {
+    it('can switch between CloudFormation and Manual setup formats', () => {
```

Comment:
the test doesn't actually test that in my opinion, it just tests that the buttons are not disabled and present. Which is already tested in the first test suite in this file